### ISSUE TITLE
Getting "any has no effect on concrete type 'Error'"

### ISSUE THREAD

**Issue by OurBigAdventure** [apple/swift-openapi-generator#580]
### Description

I'm trying to use the generator with the AppStoreConnect OpenAPI spec and I'm getting 735 instances of the error message in the title.  

### Reproduction

https://developer.apple.com/sample-code/app-store-connect/app-store-connect-openapi-specification.zip

### Package version(s)

OpenAPI 3.0.1

### Expected behavior

"any Error" doesn't cause issues even in this library so I'd expect that it would't cause issues in the generated code.

### Environment

macOS

### Additional information

I'm not able to edit most of the boxes above, sorry.  Someone may have to just download the spec and try it.

**Comment by czechboy0** [comment]
Can you share the version of Swift OpenAPI Generator you used? I just downloaded the latest App Store Connect OpenAPI doc and generated/built it just fine, so we'll need more details on what went wrong for you.

**Comment by czechboy0** [comment]
Also, what's your version of Swift, and which platform are you building on, and for?

**Comment by OurBigAdventure** [comment]
I'm on Swift 5.10 and the generator is 1.2.1 which is the latest release: https://github.com/apple/swift-openapi-generator.git

**Comment by czechboy0** [comment]
Would you be able to share a sample package that reproduces it? I tried locally and still can't see the error, thanks! 🙏

**Comment by OurBigAdventure** [comment]
interesting.  maybe I'm doing something wrong, I'm using Xcode 15.4, I'll try it with 16-beta and VS Code and see if one of those works.  I'll get back to you on what I find

**Comment by OurBigAdventure** [comment]
This is what I tried:

Started a new Vapor project (because it was easy) and added these packages

>        .package(url: "https://github.com/apple/swift-openapi-generator.git", exact: "1.2.1"),
>        .package(url: "https://github.com/apple/swift-openapi-runtime.git", exact: "1.4.0"),
>        .package(url: "https://github.com/apple/swift-openapi-urlsession.git", exact: "1.0.1"),

That built fine.

Then I started adding other packages from my project into the new clean project.  After adding all of the packages from my project all I'm getting are deprecation warnings which are expected.

So it looks like I have another error that is causing the compiler to blame this library, and this just happens to be the last thing I added before the errors appeared.  And I think caching issues preventing me from seeing this before.

I'm going to go ahead and close this out.  Thanks for testing for me, glad it turns out to be something else, I'm most excited about using this plugin and when it appeared to break I was pretty sad.
